Lib/test/test_ipaddress.py

```diff
@@ -547,6 +548,11 @@ def test_weakref(self):
         weakref.ref(self.factory('2001:db8::'))
         weakref.ref(self.factory('2001:db8::%scope'))
 
+    def test_copy(self):
+        addr = self.factory('2001:db8::%scope')
+        self.assertEqual(addr, copy.copy(addr))
+        self.assertEqual(addr, copy.deepcopy(addr))        
```

Comment:
```suggestion
        self.assertEqual(addr, copy.deepcopy(addr))
```

Reply:
Ah sorry I did not see your suggestion before I fixed it (lint bot informed me).